ui/public/locales/ja_JP.json

```
@@ -1,2367 +1,3534 @@
+"label.api.version": "APIバージョン",
+"label.apikey": "APIキー",
+"label.app.cookie": "AppCookie",
 "label.app.name": "CloudStack",
-"label.apply": "\u9069\u7528",
-"label.archive": "\u30a2\u30fc\u30ab\u30a4\u30d6",
-"label.archive.alerts": "\u30a2\u30e9\u30fc\u30c8\u306e\u30a2\u30fc\u30ab\u30a4\u30d6",
-"label.archive.events": "\u30a4\u30d9\u30f3\u30c8\u306e\u30a2\u30fc\u30ab\u30a4\u30d6",
-"label.assign": "\u5272\u308a\u5f53\u3066",
-"label.assign.instance.another": "\u307b\u304b\u306e\u30a2\u30ab\u30a6\u30f3\u30c8\u3078\u306e\u30a4\u30f3\u30b9\u30bf\u30f3\u30b9\u306e\u5272\u308a\u5f53\u3066",
-"label.assign.to.load.balancer": "\u30ed\u30fc\u30c9 \u30d0\u30e9\u30f3\u30b5\u30fc\u306b\u30a4\u30f3\u30b9\u30bf\u30f3\u30b9\u3092\u5272\u308a\u5f53\u3066\u3066\u3044\u307e\u3059",
-"label.assign.vms": "\u4eee\u60f3\u30de\u30b7\u30f3\u306e\u5272\u308a\u5f53\u3066",
-"label.associatednetwork": "\u95a2\u9023\u4ed8\u3051\u3089\u308c\u305f\u30cd\u30c3\u30c8\u30ef\u30fc\u30af",
-"label.associatednetworkid": "\u95a2\u9023\u4ed8\u3051\u3089\u308c\u305f\u30cd\u30c3\u30c8\u30ef\u30fc\u30af ID",
-"label.associatednetworkname": "\u30cd\u30c3\u30c8\u30ef\u30fc\u30af\u540d",
-"label.author.email": "\u4f5c\u6210\u8005\u306e\u96fb\u5b50\u30e1\u30fc\u30eb",
-"label.author.name": "\u4f5c\u6210\u8005\u306e\u540d\u524d",
-"label.autoscale": "\u81ea\u52d5\u30b5\u30a4\u30ba\u8a2d\u5b9a",
-"label.autoscale.configuration.wizard": "\u30aa\u30fc\u30c8\u30b9\u30b1\u30fc\u30eb\u8a2d\u5b9a\u30a6\u30a3\u30b6\u30fc\u30c9",
-"label.availability": "\u53ef\u7528\u6027",
-"label.availabilityzone": "\u30a2\u30d9\u30a4\u30e9\u30d3\u30ea\u30c6\u30a3\u30be\u30fc\u30f3",
-"label.available": "\u4f7f\u7528\u53ef\u80fd",
-"label.available.public.ips": "\u4f7f\u7528\u3067\u304d\u308b\u30d1\u30d6\u30ea\u30c3\u30af IP \u30a2\u30c9\u30ec\u30b9",
-"label.back": "\u623b\u308b",
-"label.balance": "Balance",
-"label.bandwidth": "\u5e2f\u57df\u5e45",
-"label.baremetal.dhcp.devices": "\u30d9\u30a2\u30e1\u30bf\u30eb DHCP \u30c7\u30d0\u30a4\u30b9",
-"label.baremetal.dhcp.provider": "\u30d9\u30a2\u30e1\u30bf\u30eb DHCP \u30d7\u30ed\u30d0\u30a4\u30c0\u30fc",
-"label.baremetal.pxe.device": "\u30d9\u30a2\u30e1\u30bf\u30eb PXE \u30c7\u30d0\u30a4\u30b9\u306e\u8ffd\u52a0",
-"label.baremetal.pxe.devices": "\u30d9\u30a2\u30e1\u30bf\u30eb PXE \u30c7\u30d0\u30a4\u30b9",
-"label.baremetal.pxe.provider": "\u30d9\u30a2\u30e1\u30bf\u30eb PXE \u30d7\u30ed\u30d0\u30a4\u30c0\u30fc",
-"label.baremetal.rack.configuration": "\u30d9\u30a2\u30e1\u30bf\u30eb\u30e9\u30c3\u30af\u8a2d\u5b9a",
-"label.baremetalcpu": "CPU (MHz)",
-"label.baremetalcpucores": "CPU \u30b3\u30a2\u6570",
-"label.baremetalmac": "\u30db\u30b9\u30c8\u306e MAC",
-"label.baremetalmemory": "\u30e1\u30e2\u30ea (MB)",
-"label.basic": "\u57fa\u672c",
-"label.basic.mode": "\u57fa\u672c\u30e2\u30fc\u30c9",
+"label.apply": "適用",
+"label.archive": "アーカイブ",
+"label.archive.alerts": "アラートのアーカイブ",
+"label.archive.events": "イベントのアーカイブ",
+"label.as.default": "as default",
```

デフォルトとして

---

@alkasingh03 , do you mean that 
```suggestion
"label.as.default": "デフォルトとして",
```